src/Viewer.tsx

```diff
@@ -17,58 +21,44 @@ export interface WidgetProps {
   width: number;
 }
 
-const agentColors = [
-  '#fee34d',
-  '#f7b232',
-  '#bf5736',
-  '#94a7fc',
-  '#ce8ec9',
-  '#58606c',
-  '#0ba345',
-  '#9267cb',
-  '#81dbe6',
-  '#bd7800',
-  '#bbbb99',
-  '#5b79f0',
-  '#89a500',
-  '#da8692',
-  '#418463',
-  '#9f516c',
-  '#00aabf',
-];
-
 function ViewerWidget(props: WidgetProps): JSX.Element {
+  const [modelInfo, setModelInfo] = useState<ModelInfo>({});
+
+  const handleTrajectoryData = (data: any) => {
+    setModelInfo(data.modelInfo);
+  };
+
   return (
     <div>
-      <button onClick={() => props.controller.resume()}>Play</button>
-      <button onClick={() => props.controller.pause()}>Pause</button>
-      <button onClick={() => props.controller.stop()}>stop</button>
-      <SimulariumViewer
-        renderStyle={RenderStyle.WEBGL2_PREFERRED}
-        backgroundColor={[0, 0, 0]}
-        height={props.height}
-        width={props.width}
-        loggerLevel="off"
-        onTimeChange={console.log}
-        simulariumController={props.controller}
-        onJsonDataArrived={console.log}
-        showCameraControls={false}
-        onTrajectoryFileInfoChanged={console.log}
-        selectionStateInfo={{
-          highlightedAgents: [],
-          hiddenAgents: [],
-          colorChanges: [],
-        }}
-        onUIDisplayDataChanged={(uidata) =>
-          console.log('new ui data, ', uidata)
-        }
-        loadInitialData={true}
-        hideAllAgents={false}
-        showBounds={true}
-        agentColors={agentColors}
-        showPaths={false}
-        onError={console.log}
-      />
+      <ViewerTitle modelInfo={modelInfo} />
```

Comment:
This spreads out the properties so each one is a prop instead of always having to access `modelInfo.SOMETHING]`
```suggestion
      <ViewerTitle {...modelInfo} />
```